src/components/PeopleAndRoles/CurrentDirectors.vue

```diff
@@ -88,21 +87,16 @@ export default class CurrentDirectors extends Mixins(CommonMixin) {
   /** Headers for the person table. */
   readonly tableHeaders = ['Name', 'Mailing Address', 'Delivery Address', 'Effective Dates']
 
-  /**
-   * Called when component is mounted.
-   */
-  private mounted (): void {}
-
   /**
    * Returns the current directors from the people and roles.
    * @returns array of directors
    */
   private get currentDirectors () : OrgPersonIF[] {
     const directors = this.getPeopleAndRoles
       .filter(people => people.roles.some(role =>
-        role.roleType.toLowerCase() === 'director' && role.cessationDate === null))
+        role.roleType.toLowerCase() === RoleTypes.DIRECTOR.toLowerCase() && !role.cessationDate))
```

Comment:
I am still not OK with this.

How did `roleType` get assigned and why is it not the same value (string) as the RoleTypes enum?

Does it come from the API, and need to be converted to the enum when the data is fetched?

Reply:
This was the easiest way to make the comparison case insensitive. I'll take a look at the data and see if it's just the test data that is different. I didn't want to touch the enum itself because I figured that might break something

Reply:
The comparison should just work like this:
`role.roleType === RoleTypes.DIRECTOR`

... at least, as long at `role.roleType` is properly declared as type RolesTypes and wasn't assigned a value that isn't of that type.

Reply:
That was the first thing I did when I started working on it but at least for the business I've been testing with that doesn't work. The field is typed as a RoleType but the content doesn't match that. Just investigating now to see where that data is coming from to see

Reply:
Thanks for the fix.